ciphers/trifid_cipher.py

```diff
@@ -1,8 +1,28 @@
-# https://en.wikipedia.org/wiki/Trifid_cipher
+"""
+The trifid cipher uses a table to fractionate each plaintext letter into a
+trigram,mixes the constituents of the trigrams, and then applies the table
+in reverse to turn these mixed trigrams into ciphertext letters.
+https://en.wikipedia.org/wiki/Trifid_cipher
+"""
+
 from __future__ import annotations
 
 
 def __encrypt_part(message_part: str, character_to_number: dict[str, str]) -> str:
+    """
+    Arranges the triagram value of each letter of 'message_part' vertically
+    and joins them horizontally

```

Comment:
Use _active_ tense like you are commanding the computer what to do.
```suggestion
    Arrange the triagram value of each letter of 'message_part' vertically and join them
    horizontally.
```